client/extensions/zoninator/components/settings/zones-dashboard/zone-item.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { find, get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import sectionsModule from 'sections';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+const ZoneItem = ( { siteSlug, zone } ) => {
```

Comment:
There already is a `Zone` component in the extension - it's the one that's used to render zone settings. I thought it might be a good idea to keep the names from overlapping as much as possible.